Install instuctions added

Sozialarchiv opened the pull request.

Commit:
Install instuctions added

kerim1 commented:
The part about Docker Compose does not make sense as there is no compose file. And you are missing the `IIIF_IMAGE_CONCURRENCY` parameter, which controls the number of threads libvips can use for image processing. By default it takes the number of CPU cores available. Or you specify 0 to make this explicit.

Commit:
Docker Compose text removed

Commit:
IIIF_IMAGE_CONCURRENCY to configuration doc added

Sozialarchiv commented:
> The part about Docker Compose does not make sense as there is no compose file. 

I removed it.

>  And you are missing the IIIF_IMAGE_CONCURRENCY parameter, which controls the number of threads libvips can use for image processing. By default it takes the number of CPU cores available. Or you specify 0 to make this explicit.

I added it to the doc.

kerim1 commented on readme.md:
```diff
@@ -17,14 +17,6 @@ E.g. http://localhost:3333/example.jpg/full/!100,100/0/default.jpg
 
 Use the provided Docker Compose or install manually.
```
Missed this part about Docker Compose

kerim1 reviewed (commented):
Just the one last thing about Docker Compose to be removed and then it can be merged.

Commit:
Docker removed in installation doc
